packages/react/src/components/UIShell/HeaderMenu.js

```diff
@@ -180,6 +191,7 @@ class HeaderMenu extends React.Component {
       focusRef, // eslint-disable-line no-unused-vars
       ...rest
     } = this.props;
+    let { isActive } = this.props;
```

Comment:
```suggestion
```

Reply:
I would add this above `isCurrentPage` in the above section.

Reply:
The above props and the `isCurrentPage` prop are being destructured as `const`, so we won't be able to infer the value to `isActive` from `isCurrentPage` as we cannot re-assign value to const.
As we will remove the `isCurrentPage` in the next major release, then only we can import it along with other props. I guess, we will need this let variable till then.

Reply:
You need to destructure them both, and then assign them to a new `let` variable further down: `let isActivePage = isActive ? isActive : isCurrentPage;` and then use `isActivePage` handle all the `isActive/`isCurrentPage` functionality.